### PR TITLE
Added Warning for CMv2 Update Number

### PR DESCRIPTION
**Pull request by stegaBOB**

### CODE REVIEW

File: docs/candy-machine-v2/09-update.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ sidebar_label: "7. Update the Candy Machine"
 ---
 # Update the Candy Machine
 
-**Any** configuration setting can be updated in the `CMv2` with a single command.
+**Any** configuration setting (besides `number`) can be updated in the `CMv2` with a single command.
```
[the remark this answers is not in the document]
Comment (stegaBOB) [reply docs/candy-machine-v2/09-update.md:6]: Definitely better lol. Are you able to request the change with the review thing?

File: docs/candy-machine-v1/01-introduction.md [hunk 2]
```diff
@@ -19,4 +19,4 @@ After addressing the [prerequisites](#prerequisites), documented below, the gene
 
 :::
 
-As you are no longer able to create new CMv1s, this guide will assume you have already created one and are operating a CMv1 drop on the mainnet-beta cluster.
\ No newline at end of file
+As you are no longer able to create new `CMv1`s, this guide will assume you have already created one and are operating a `CMv1` drop on the mainnet-beta cluster.
```
[the remark this answers is not in the document]
Comment (stegaBOB) [reply docs/candy-machine-v1/01-introduction.md:22]: That was intentional on my end. Shouldn't it be outside?

File: docs/candy-machine-v2/09-update.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (roederw) [reply docs/candy-machine-v2/09-update.md:6]: Probably. I don't know how to Github. Wen Azure DevOps?

[the remark this answers is not in the document]
Comment (roederw) [reply docs/candy-machine-v2/09-update.md:6]: Let me try.

Comment (roederw) [inline docs/candy-machine-v2/09-update.md:6]:
```suggestion
**Any** configuration setting, with the exception of `number`, can be updated in a `CMv2` with a single command.
```

Comment (roederw) [reply docs/candy-machine-v2/09-update.md]: ooooo

[the remark this answers is not in the document]
Comment (stegaBOB) [reply docs/candy-machine-v2/09-update.md]: > **Any** configuration setting, with the exception of `number`, can be updated in the `CMv2` with a single command. Can I edit this?
> 
> Sammy is the best

Thanks will!

**Review by roederw — APPROVED** [review]

**Pull request merged**